python/pyADflow.py

```diff
@@ -3298,8 +3298,11 @@ def solveAdjoint(self, aeroProblem, objective, forcePoints=None,
             # Now set the flags and possibly reset adjoint
             if self.adflow.killsignals.adjointfailed:
```

Comment:
" though the adjoints after a failed one get solved, they still get reset later."

If the adjoints after a failed one are solved then wouldn't they jump to the else statement and set the adjoint = psi

Reply:
> " though the adjoints after a failed one get solved, they still get reset later."
> 
> If the adjoints after a failed one are solved then wouldn't they jump to the else statement and set the adjoint = psi

I realized the phrasing is a bit confusing, so when I mentioned "solved", it just means that we "solved" the adjoint `self.adflow.adjointapi.solveadjoint(RHS, psi, True)`, but does not necessarily mean that the adjoint "converged". So the statement I had there " though the adjoints after a failed one get solved, they still get reset later.", it basically refer to those cases that adjoint reaches the maximum iteration but does not converge to the specified tolerance. In that case, the `killsignals.adjointfailed` is still True, so then adjoint gets reset `self.curAP.adflowData.adjoints[objective][:] = 0.0`, which is not what we expect when we use `skipafterfailedadjoint: True`.